src/script/engine.zig

```diff
@@ -265,6 +267,24 @@ pub const Engine = struct {
         return error.EarlyReturn;
     }
 
+    /// OP_TOALTSTACK: Puts the value onto the top of the alt stack, and removes it from the main stack.
+    ///
+    /// # Returns
+    /// - `EngineError`: If an error occurs during execution
+    fn opToAltStack(self: *Engine) !void {
+        const value = try self.stack.pop();
+        try self.alt_stack.pushElement(value);
+    }
+    
+    /// OP_FROMALTSTACK: Puts the value onto the top of the main stack, and removes it from the alt stack.
+    ///
+    /// # Returns
+    /// - `EngineError`: If an error occurs during execution
```

Comment:
rather than this comment, write the error type in the function signature:
`EngineError!void`

it will give the same information to the user, more to the compiler, and the language server allow you to jump to the error type

Reply:
@tdelabro makes sense, I added `EngineError!void`